src/overlays/actors/ovl_En_Mushi2/z_en_mushi2.c

```diff
@@ -1,3 +1,10 @@
+/*
+ * File: z_en_mushi2.c
+ * Overlay: ovl_En_Mushi2
+ * Description: Bugs
```

Comment:
This was mentioned in discord, but there are two types of bugs. It'd be good to clarify which it is, although you'd have to double check over in discord which is which:
https://discord.com/channels/688807550715560050/688851976746041391/890153066299539466

Reply:
```suggestion
 * Description: Three bugs that burrow
```